### ISSUE TITLE
PHP8 incompatible ?

### ISSUE THREAD

**Issue by n9yty** [WPFreighter/wp-freighter#6]
With PHP7.4 EOL'd, we had moved to PHP8. Trying out a fresh install of WP-Freighter does not work. In the GUI I see no action with clicking create or clone the site, but in the error log I see this:

```
PHP Warning:  Undefined array key "value" in /home/newsite/public_html/wp-content/plugins/wp-freighter/app/Run.php on line 37
PHP Fatal error:  Uncaught ValueError: max(): Argument #1 ($value) must contain at least one element in /home/newsite/public_html
/wp-content/plugins/wp-freighter/app/Run.php:186
Stack trace:
#0 /home/newsite/public_html/wp-content/plugins/wp-freighter/app/Run.php(186): max(Array)
#1 /home/newsite/public_html/wp-includes/class-wp-hook.php(308): WPFreighter\Run->ajax_actions('')
#2 /home/newsite/public_html/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters('', Array)
#3 /home/newsite/public_html/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#4 /home/newsite/public_html/wp-admin/admin-ajax.php(188): do_action('wp_ajax_stacked...')
#5 {main}
thrown in /home/newsite/public_html/wp-content/plugins/wp-freighter/app/Run.php on line 186
```

**Comment by cvladan** [comment]
Is this fixed? Is it working on PHP8?

**Comment by austinginder** [comment]
@n9yty @cvladan My main WP Freighter instance is running stable on PHP 8 however the sites were added prior to the PHP 8 update. Looks like there are a number of PHP 8 bugs to squash. I did a round of development and [released v1.1.1](https://github.com/WPFreighter/wp-freighter/releases/tag/v1.1.1). That might not address all of the PHP 8 issues however from my testing it does resolve this one. Give it a try now and thanks for the report.
